pkg/cnb/image_rebaser_test.go

```diff
@@ -0,0 +1,143 @@
+package cnb_test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/buildpack/imgutil/fakes"
+	"github.com/buildpack/lifecycle"
+	"github.com/buildpack/lifecycle/logging"
+	"github.com/sclevine/spec"
+	"github.com/stretchr/testify/require"
+	"github.com/tj/assert"
```

Comment:
Did you mean to use this assert?

Reply:
good catch. to answer the question, hell no